### PR TITLE
Bug 1428285 Add NTP as a prerequisite

### PR DESCRIPTION
**Pull request by bmcelvee**
https://bugzilla.redhat.com/show_bug.cgi?id=1428285

### CODE REVIEW

**Comment by bmcelvee** [comment]
@ahardin-rh @mburke5678 peer review please! 😺

**Comment by mburke5678** [comment]
LGTM. 
One nit, do we want to get rid of the passive? 
"Network Time Protocol (NTP) must be enabled"
"You must enable Network Time Protocol..."

**Comment by bmcelvee** [comment]
Ah yes, thank you @mburke5678

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -192,6 +192,18 @@ SELINUX=enforcing
 SELINUXTYPE=targeted
 ----
 
+[[prereq-NTP]]
+=== NTP
+
+You must enable Network Time Protocol (NTP) to prevent masters and nodes in the
+cluster from going out of sync. Set `openshift_clock_enabled` to true in the
```
Comment (ahardin-rh) [inline install_config/install/prerequisites.adoc]: `true`

**Comment by ahardin-rh** [comment]
@bmcelvee just one comment from me. Otherwise, looks good!

**Comment by bmcelvee** [comment]
Thanks @ahardin-rh!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../install_config/install/prerequisites.adoc#install-config-install-prerequisites[Installing a Cluster -> Prerequisites]
|Added xref:../install_config/install/prerequisites.adoc#prereq-NTP[Network Time Protocol (NTP)] as a prerequisite.
%

**Pull request merged**